activation/1.2/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Jakarta Activation 1.2"
+date: 2019-10-01
+summary: "Initial release of Jakarta Activation"
+---
+Jakarta Activation defines a set of standard services to: determine the
+MIME type of an arbitrary piece of data; encapsulate access to it;
+discover the operations available on it; and instantiate the
+appropriate bean to perform the operation(s).
+
+* [Jakarta Activation 1.2 Specification Document](./activation-spec_1.2.pdf) (PDF)
```

Comment:
The actual files have the correct names, but these references do not.  There should be a dash "spec-1.2", not an underscore "spec_1.2".

Reply:
Fixed.